github/repos.go

```diff
@@ -1212,6 +1234,12 @@ type SignaturesProtectedBranch struct {
 	Enabled *bool `json:"enabled,omitempty"`
 }
 
+// AutomatedSecurityFixes represents their status.
+type AutomatedSecurityFixes struct {
+	Enabled *bool `json:"enabled"`
+	Paused  *bool `json:"paused"`
```

Comment:
```suggestion
	Enabled *bool `json:"enabled,omitempty"`
	Paused  *bool `json:"paused,omitempty"`
```

Reply:
@gmlewis could you explain this to me so I understand it's purpose in any future PR I do?

Reply:
Please let me know if https://github.com/google/go-github/pull/2827#issuecomment-1621990607 answers your question.

Reply:
Thanks for the link @gmlewis it makes it very clear. However, having read that I think my original form was possibly correct. Looking at the response schema in the API documentation it states that both fields are required in the response, so if I understood the link correctly these should not have omitempty set?

Reply:
Ah, then I apologize.  If both are required, then you are right - not only does `omitempty` not need to be added, but these fields also do not need to be pointers.  They can simply be `bool`.

Reply:
let me update the PR

Reply:
Historically, we have erred on the side of using pointers since sometimes GitHub responses have included things in some API calls and omitted them in others.  But if these will always be present, then there is no need to make them pointers.  

However, I'm fine if you just want to drop the `omitempty` and revert to your original version.

Reply:
Pushed the changes to remove the omitempty.